xarray/__init__.py

```diff
@@ -65,10 +65,27 @@
 # A hardcoded __all__ variable is necessary to appease
 # `mypy --strict` running in projects that import xarray.
 __all__ = (
-    # Sub-packages
-    "groupers",
-    "testing",
-    "tutorial",
+    "ALL_DIMS",
+    # Classes
+    "CFTimeIndex",
+    "Context",
+    "Coordinates",
+    "DataArray",
+    "DataTree",
+    "Dataset",
+    "Index",
+    "IndexSelResult",
+    "IndexVariable",
+    # Exceptions
+    "InvalidTreeError",
+    "MergeError",
+    "NamedArray",
+    "NotFoundInTreeError",
+    "SerializationWarning",
+    "TreeIsomorphismError",
+    "Variable",
```

Comment:
The sorting of this list is wrong now. `Variable` is not an Error.

Reply:
yeah, I don't think we want to alphabetically sort that list if we want to keep the sections

Reply:
Yes, let's keep this list as-is

Reply:
@DimitriPapadopoulos if you want to pass feedback upstream — the sorting should split on comments, so Sections work

Reply:
It feels like a very specialised need. In this case, I will revert and add a `noqa`.

Reply:
Actually, `noqa` is not needed yet. This is a "preview" rule.

Reply:
> It feels like a very specialised need

FWIW I think this is fairly common in code formatters. I think rust imports are done this way, for example...